### DISCUSSION TITLE
commitMessagePrefix does not have a colon appended to it when generating commits, but the documentation says it should

### DISCUSSION THREAD

**Discussion by pgvsalamander** [renovatebot/renovate#27257] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.182.1

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

Per https://docs.renovatebot.com/configuration-options/#commitmessageprefix:
> Renovate always appends a `:` after the commitMessagePrefix. For example, if you set commitMessagePrefix to `chore`, Renovate turns it into `chore:`.

I set `commitMessagePrefix` to `build`, and the bot generated the following commit: `build Update dependency typescript to v4.9.5`

I can create a repro repo if this is a bug rather than a documentation issue.

Full config:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "assignees": ["redacted"],
  "labels": ["test label"],
  "prHourlyLimit": 5,
  "commitMessagePrefix": "build"
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```



```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
This is a docs error. PRs welcome to remove/correct that statement

**Reply by pgvsalamander** [reply]
https://github.com/renovatebot/renovate/pull/27258
